tests/mamba/selective_state_update_triton.py

```
@@ -280,9 +398,14 @@ def selective_state_update_triton(
         assert dt_bias.shape == (nheads, dim)
     if state_batch_indices is not None:
         assert state_batch_indices.shape == (batch,)
-    out = torch.empty_like(x)
+    assert out.shape == x.shape
+
     grid = lambda META: (triton.cdiv(dim, META["BLOCK_SIZE_M"]), batch, nheads)
```

Ruff (0.14.14)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `grid` as a `def`

(E731)